samples/javascript_nodejs/51.teams-messaging-extensions-action/bots/teamsMessagingExtensionsActionBot.js

```diff
@@ -164,4 +173,57 @@ function shareMessageCommand(context, action) {
     };
 }
 
+function empDetails(context, action)
+{
+    return {
+        task: {
+          type: 'continue',
+          value: {
+            width: 350,
+            height: 300,
+            title: 'Task module WebView',
+            url: `${baseurl}/CustomForm.html`,
+          }
+        }
+    };
+}
+
+function dateTimeInfo(context, action)
+{
+    return {
+        task: {
+          type: 'continue',
+          value: {
+            width: 450,
+            height: 125,
+            title: 'Task module Static HTML',
+            url: `${baseurl}/StaticPage.html`,
+          }
+        }
+    };
+}
```

Comment:
Since neither of these functions use the arguements, `context` or `action`, you can remove them both here and above in the `switch/case` statement.